crates/starknet-devnet-server/src/api/http/models.rs

```diff
@@ -52,14 +52,20 @@ pub struct AbortedBlocks {
 }
 
 #[derive(Deserialize)]
-pub struct Time {
+pub struct IncreaseTime {
     pub time: u64,
```

Comment:
Why shouldn't `IncreaseTime` also have optional block generation?

Reply:
the requested feature was about set time so I didn't implement it in increase time

increase time is only changing time shift so it wouldn't help our user who expect the exact time in the next block

Reply:
Ok, I guess it makes sense. Resolving.